Lab2.ipynb

```diff
@@ -0,0 +1,830 @@
+      "text/plain": [
+       "<Figure size 720x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(1, 1)\n",
+    "\n",
+    "x = np.linspace(stats.rayleigh.ppf(0.01),stats.rayleigh.ppf(0.99), 100)\n",
+    "d1 = stats.rayleigh.pdf(x)\n",
+    "d2 = stats.rayleigh.pdf(x)\n",
+    "\n",
+    "n = 5\n",
+    "for i in range(1, n):\n",
+    "    \n",
+    "    dist_5 = np.convolve(d1,d2)\n",
+    "    d1 =  dist_5\n",
+    "\n",
+    "dist_5 = dist_5/dist_5.sum()\n",
+    "#plt.plot(dist_5);\n",
+    "left, right = plt.xlim()\n",
+    "x_sum = np.linspace(left, right, dist_5.size)\n",
+    "cx = x_sum/n\n",
+    "\n",
+    "plt.plot(cx,dist_5);\n",
```

Comment:
I am not quite sure if for this plot and the next one you are plotting the sum of distribution for 5 and 100 days or you are plotting the average over 5 and 100 days. From your code and what problem 2A asks for I think you are trying to plot the average. But your x-axis labels are "sum". 
Also, if you are doing averages, I believe the range of the x-axis for average plots should be the same as the range of the x-axis in the original distribution you are using to take the average. I think the first plot you have in this section Q2A is the original Rayleigh distribution you are using to do convolution and then take averages. That plot is from 0 to 3. For example, if you want to average over 5 days. Then first you need to convolve the original plot 5 times, then your plot for the sum of 5 days might have a x-axis from 0 to 3*5 = 15. And when you average the sum of 5 days, we want to rescale the x-axis, divide the x-axis for the sum plot by number of convolution(5), then the x range for the average plot should be from 0 to 15/5 = 3 which is the same with the x range for the original Rayleigh plot.